server/server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const port = 3002;
+const app = express();
+
+app.use(bodyParser.json());
+
```

Comment:
"app.use(express.static(__dirname + '/../public/dist'));"

Adding the above line would be useful here in order to make sure express will serve up your static files.

Reply:
In addition, you would need the link in the get request to be '/amenities' or anything not '/' because the express.static will automatically respond to a get request on '/' with the static file.

Reply:
Thanks, these are helpful suggestions!